tools/livelog/main.go

```diff
@@ -168,7 +168,11 @@ func main() {
 
 	runServer = func(server *http.Server, addr, crtFile, keyFile string) error {
 		server.Addr = addr
-		return server.ListenAndServe()
+		if crtFile != "" && keyFile != "" {
+			return server.ListenAndServeTLS(crtFile, keyFile)
+		} else {
+			return server.ListenAndServe()
+		}
```

Comment:
Looks like I blew this up in https://github.com/taskcluster/taskcluster/commit/fa6d273984 -- sorry about that!

I think we didn't notice until now because (a) generic-worker doesn't use the TLS support in livelog and (b) docker-worker was stuck on an ancient version.